includes/wikia/Wikia.php

```diff
@@ -85,6 +85,7 @@ class Wikia {
 
 	const COMMUNITY_WIKI_ID = 177; // community.wikia.com
 	const NEWSLETTER_WIKI_ID = 223496; // wikianewsletter.wikia.com
```

Comment:
This wiki has one page only oO

Reply:
But way more [images](http://wikianewsletter.wikia.com/wiki/Special:Images)

Reply:
So it's just a image storage with no other use? They could be stored elsewhere.